crates/nu-command/src/viewers/table.rs

```diff
@@ -22,10 +22,10 @@ const INDEX_COLUMN_NAME: &str = "index";
 fn get_width_param(width_param: Option<i64>) -> usize {
     if let Some(col) = width_param {
         col as usize
-    } else if let Some((Width(w), Height(_h))) = terminal_size::terminal_size() {
-        (w - 1) as usize
+    } else if let Some((Width(w), Height(_))) = terminal_size::terminal_size() {
+        w as usize
```

Comment:
This is interesting. I remember a change to specifically subtracting 1 from w because returning w broke something somewhere. Let's see what happens I guess.

Reply:
Found it, this is what I was thinking of https://github.com/nushell/nushell/pull/2646

Reply:
Actually it may break something....
Because as I've showed in wrap creates additional char sometimes.

So I can imagine some case where because of it the layout will be broken. 

So it may be better not merged unless wrap will be fixed.

Reply:
any ideas how to fix wrap yet?

Reply:
https://github.com/elferherrera/nushell/commit/96bdcc4ff73211043ab6900ee9fb593c3f598b27

Reply:
That looks familiar. Looks like it was in engine-q when that change was made. oh, ya. it was in engine-q https://github.com/nushell/engine-q/pull/346

Reply:
Maybe `w` is fine now with tabled drawing nushell tables now? I wonder how we can test whether we need `w` versus `w - 1`?

Reply:
oh oh oh. I remember no. with `w` the display and tables were wrapping to another line. so the line of the right side of the table would wrap all the way passed on the right side and would show up on the left side. That's why we changed it to -1.